site/guide/set-up-model-workflows.qmd

```diff
@@ -0,0 +1,213 @@
+
+::: {.callout-tip title="Example" appearance="minimal"}
+A `Put On Hold` action that requests a value in the Date On Hold field, and transitions the resource status to On Hold.
+:::
+
+:::
+
+:::
+
+#### Approval
+- When your resource reaches the approval step, users in the selected Approval Group will receive your configured message. 
+- Requires a User type [custom field](manage-inventory-custom-fields.qmd) to select from, and must be connected to both an approval and rejection **[Status Change](#status-change)** step.
+
+::: callout 
+Please note that users must be assigned to the Approval Group custom field before it is transitioned into the **Approval** workflow step in order for the message to be displayed to them. 
+:::
+
+| Field | Description |
+|---:|---|
+| Approval group (required) | Select the User type custom field that indicates users responsible for approval. |
+| Threshold (required) | Percentage of approvers required for the resource to be approved. |
+| Approval Message (required) | When your resource reaches this approval step, the users in the selected group will recieve the following message. |
+: Approval Group step configuration {.hover tbl-colwidths="[35,65]"}
+
+![](approvals-message.png){fig-alt="A screenshot showing the approval message shown to the Approval Group."}
+
+::: {.grid}
+
+::: {.g-col-7}
+![](approval-group-setup.png){fig-alt="A screenshot showing an Approval Group step configuration with a custom field selection of Approval Users."}
+:::
+
+::: {.g-col-5}
+
+::: {.callout-tip title="Example" appearance="minimal"}
+A message shown to the Approval Group users requesting approval of model documentation.
+:::
+
+:::
+
+:::
+
+### Add conditional requirements
+
+Conditional requirements can be configured for all three available step types:
+
+| Step type | Conditional options |
+|---:|---|
+| Status Change | Under `When these conditions are met`, you are able to set both `AND` and `OR` conditions. |
+| User Action | Under `Display action button when`, you are able to set both `AND` and `OR` conditions.  |
+| Approval | Under `Threshold`, you are able to set the minimum percentage of approvers required for the resource to be approved. |
+: Step type conditional options {.hover tbl-colwidths="[20,80]"}
+
+For [Status Change](#status-change) and [User Action](#user-action) conditions, you're able to add a single independent **Rule** or a linked condition **Group**. These rules and groups can be nested if desired:
+
+- Click **+ Rule** to add an independent rule.
+- Click **+ Group** to add a linked group of rules that all must be true to qualify. 
+
+
+## Link workflow together
+
+To initiate the beginning of your workflow:
+
+1. Locate the **Start** of your workflow.
+
+2. Drag from the **.** at the bottom of **Start** to the top **{{< fa circle >}}** on the first step of your workflow.
```

Comment:
```suggestion
2. Drag from the period (**.**) at the bottom of **Start** to the top **{{< fa circle >}}** on the first step of your workflow.
```

This read like a broken sentence until I clued in. Not ideal, but I wonder if spelling it out isn't preferrable here?